k8s/tf/85_asg_monitor/eu-central-1/main.tf

```diff
@@ -0,0 +1,22 @@
+provider "aws" {
+  region  = "eu-centraul-1"
+  version = "~> 2"
+}
+
+terraform {
+  backend "s3" {
+    bucket = "sumo-state-095732026120"
+    key    = "terraform/sumo-infra-sns-asg-eu"
+    region = "us-west-2"
+  }
+}
+
+variable "slack_webhook_url" {}
+
+module "monitor_asgs" {
+  source = "github.com/mozilla-it/asg-alert?ref=tags/1.0"
+  slack_webhook_url = "${var.slack_webhook_url}"
+  slack_channel     = "it-sre-bot"
+  slack_username    = "AWS"
+  asgs              = ["nodes.k8s.eu-central-1a.sumo.mozit.cloud", "master-eu-central-1a.masters.k8s.eu-central-1a.sumo.mozit.cloud"]
```

Comment:
can you separate these on individual lines? if so, i prefer that because of readability and git diffs based upon a single line.

Reply:
Made asgs variable multi-line, thanks.